Form-Controls/styles.css

```diff
@@ -0,0 +1,31 @@
+body {font-family: Arial, Helvetica, sans-serif;}
+* {box-sizing: border-box;}
+
+input[type=text],input[type=email] select, fieldset{
```

Comment:
Does this rule apply to all the elements you intended? Take another look at the selectors